masterlist.yaml

```
@@ -5012,6 +5012,74 @@ plugins:
   - name: 'Project Nevada - Med-X Fix.esp'
     tag: [ Names ]
 
+# Mothership Zeta Rehaul
+  - name: '(Mothership)?ZetaRehaul((BiWastelands|DelayDLCRedux|HardcoreZeta|HardcoreZetaDelayDLCRedux|NVSpeechChecks)Patch)?\.es[mp]'
+    url:
+      - link: 'https://www.nexusmods.com/newvegas/mods/77340/'
+        name: 'TTW - Mothership Zeta Rehaul'
```

Removed comment as we already have the mod name listed below.
I would only add header comments in cases where a mod has a lot of plugins.
And you want to groups them by type.

Changed the regex a bit to make it as simple as possible.
I would aim for clarity first, for whoever is going to be reading it in future.
Rather than trying to make it as short as possible.

```suggestion
  - name: '(MothershipZetaRehaul|ZetaRehaul(BiWastelands|DelayDLCRedux|HardcoreZeta|HardcoreZetaDelayDLCRedux|NVSpeechChecks)Patch\.es[mp]'
    url:
      - link: 'https://www.nexusmods.com/newvegas/mods/77340/'
        name: 'TTW - Mothership Zeta Rehaul'
```